dlp/dlp_snippets/main.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // dlp is an example of using the DLP API.
+
```

Comment:
Please leave this newline in -- it serves as the package documentation. See https://github.com/golang/go/wiki/CodeReviewComments#package-comments.

Reply:
Oops, this was unintended. I will revert it.